### ISSUE TITLE
No parameters on Steel Connection objects Revit 2021

### ISSUE THREAD

**Issue by BIMLUND** [Autodesk/revit-ifc#348]
Hi!
Thanks for fixing the export of steel connections! Great! 
I have a problem however. I cannot get any parameters to export with the steel connection elements.
Not any custom user defined property set, and not any Revit property set. 

Versions:
![image](https://user-images.githubusercontent.com/81314195/133976936-b9c433f1-c839-4c9f-a221-7a635b11b4e1.png)

Is this an error or am I doing something wrong? The other elements (steel columns etc) have all properties. Please see the screenshot.

All properties:
![image](https://user-images.githubusercontent.com/81314195/133976689-39100970-ee8b-415b-aa03-b0df96afc16c.png)

No properties on plate (same IFC):
![image](https://user-images.githubusercontent.com/81314195/133976821-375ab7c5-2b33-48a6-8db8-9a0c262ef0b9.png)

**Comment by BIMLUND** [comment]
I downloaded IFC export 21.4.2.0 but it didnt help.

I cannot override the mapping of Steel connections either. Thought that if I export it as "IfcBuildingElementProxy" it might work.

But all Steel connections come out as ifcPlate, IfcMechanicalFastener etc. And that is great! But if we want to override it for some reason it is not possible.

This did not work:
![image](https://user-images.githubusercontent.com/81314195/134509832-5d09c3d1-3052-4e52-b9de-a8512a0c9b4f.png)

![image](https://user-images.githubusercontent.com/81314195/134510072-d78ccc93-684c-4c03-8c6a-c8cffe32cddd.png)

**Comment by BIMLUND** [comment]
Attached are sample files. Revit 2021 and an IFC-export (2x3)
[Steel revit+ifc.zip](https://github.com/Autodesk/revit-ifc/files/7217864/Steel.revit%2Bifc.zip)

**Comment by BIMLUND** [comment]
Is this an issue that will be addressed?

**Comment by BIMLUND** [comment]
It is not working on 2022 with the latest IFC exporter either. No interest in fixing this issue? We cannot do QTO on steel because we don't even can export the parameters Length, Width and Thickness. Thanks!

**Comment by BIMLUND** [comment]
@AngelVelezSosa

**Comment by COWImgbj** [comment]
@AngelVelezSosa Any progress on this? 

This seems to only be affecting the Fabrication Elements(bolts, plates etc.), and not the actual structural connection element. They export as IfcBuildingElementProxy with psets, IfcGUID and BATID, but any plates/boltes hosted inside the connection element as subelements or modelled standalone are not exported correctly.

![image](https://user-images.githubusercontent.com/58807838/182640435-ed48b251-8fb3-4fa1-bbd1-c955699113ab.png)

In earlier versions, I think 2020 or 2021, it was possible to export just the structural connection element without bolts/plates as subelements. In 2022 this doesn't seem possible anymore. Now we get duplicate elements. It would be nice to have an option to export just the main structural connection element or the subelements, and not both. But first, the subelements needs to be exported correctly.

**Comment by AngelVelezSosa** [comment]
Let me check on the status of this.

**Comment by BIMLUND** [comment]
I was messing around and found a workaround in Revit 2022.1 at least.
Tried IFC-export: 22.5.2.0

Weirdly: Do NOT check "Include steel elements". 
![2022-09-13 09_33_06-Modify Setup](https://user-images.githubusercontent.com/81314195/189839115-f34eab56-40af-4676-95c2-778f1b0b2235.png)

Set the IFC mapping table to export Proxyelements of all Structural connections:
![elemnt](https://user-images.githubusercontent.com/81314195/189838885-48fdf01b-ab3a-45f6-94b6-c5c7439ca9ac.png)

And then it works...

**Comment by BIMLUND** [comment]
> Let me check on the status of this.

My "workaround" dosent work anymore. Not sure why. 
Any progress in getting this to work? Parameters on Steel connections? @AngelVelezSosa

**Comment by AngelVelezSosa** [comment]
Yes, we fixed some steel connections export issues, but the parameters issues needs a fix in some external code that isn't part of the open source.  It's been identified and assigned to the appropriate team as an open issue.

**Comment by winipeti** [comment]
Hi! Is there any progress about solving the issue? I have tons of plates in 5 project and desperate to export their parameters :) 
Thanks!

Peter
